### DISCUSSION TITLE
V7 is incredible

### DISCUSSION THREAD

**Discussion by keanugrieves** [mantinedev/mantine#4686] [Kind words]
Oh, where to begin...

Recently, I made the leap to Remix from my beloved Nuxt, and after experimenting with a few UI frameworks, I landed on Mantine V6, watching with feverish anticipation as the V7 alpha took shape.

But now that the beta is out, I'm building a pair of linked projects (a massive admin dashboard, along with a client-facing site) and thanks to the emphasis on setting defaults and styling through the provider, along with the jaw-dropping variety of components and hooks, I've never worked faster, slapping together these two data-intensive, genuinely responsive SSR projects in record time without getting bogged down in the style-fussing that's eaten up so much of my time in the past.

Every component I've imported is pretty out of the box, and the structure of this project is so clean, well-designed, well-separated and well-considered, I can build two very different-looking sites while still sharing code between them.

Hats off and big props because this genuinely feels like magic, as Mantine V7 makes development a blast, a breeze and immediately rewarding. Between this and [ZenStack](https://zenstack.dev), 2023 has proven to be an embarrassment of riches, and we still have three months to go.

**Comment by rtivital** [comment]
Thanks for kind words!

**Comment by sarendipitee** [comment]
I whole heartedly agree! I've used just about every UI library under the sun and Mantine is by far the best and most useful. It has a delicate balance of doing enough for the developer while also maintaining flexibility. The out of the box theme is just plain gorgeous too.

V7 is also an amazing direction to head. I've never been a fan of the CSS-in-JS movement. It always seemed silly to waste so many cpu cycles parsing and inserting styles on the render thread when CSS has had that problem solved for 20+ years. So yea I'm loving V7 so far and plan on helping getting it polished up for release.

**Comment by murbanowicz** [comment]
Is there any docs available for v7? I'd love to use it in new project. 

Unless it's not that different in usage from v6?

**Reply by rtivital** [reply]
https://v7.mantine.dev/

**Reply by murbanowicz** [reply]
Thank you!

**Comment by thecmdrunner** [comment]
🫶 Still can't find a sponsor button! @rtivital

**Reply by Uzaaft** [reply]
Put some effort into reearching your question: https://github.com/orgs/mantinedev/discussions/1061
